docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

```diff
@@ -1,11 +1,40 @@
 # Converting a PyTorch Model {#openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch}
 
-The PyTorch framework is supported through export to the ONNX format. In order to optimize and deploy a model that was trained with it:
+The PyTorch framework is supported through export to the ONNX format. Model Optimizer Python API allows to convert PyTorch models through usage of convert_model() method which internally converts a model to ONNX.
+## Conversion of PyTorch model from the memory using Python API (Experimental Functionality)
+Converting a PyTorch model using convert_model() requires providing of 'input_shape' or 'example_input'.
+```sh
+import torchvision
+import torch
+from openvino.tools.mo import convert_model
+
+model = torchvision.models.resnet50(pretrained=True)
+ov_model = convert_model(model, example_input=torch.zeros(1, 3, 100, 100))
+```
+
+'example_input' accepts following formats:
+
+* openvino.runtime.Tensor
+* torch.Tensor
+* np.ndarray
+* list or tuple with tensors (openvino.runtime.Tensor / torch.Tensor / np.ndarray)
+* dictionary where key is input name, value is tensor (openvino.runtime.Tensor / torch.Tensor / np.ndarray)
+
+ONNX opset version can be set using 'onnx_opset_version' parameter.
```

Comment:
is it optional? If yes, please provide info which value is default

Reply:
Yes, it is optional. Added "optional" note and default behavior description.